src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -74,6 +75,10 @@ const MeasureEditor = () => {
   // error markers control the error underlining in the editor.
   const [errorMarkers, setErrorMarkers] = useState<EditorErrorMarker[]>([]);
 
+  const { getUserName } = useOktaTokens();
+  const userName = getUserName();
+  const canEdit = userName === measure.createdBy ? true : false;
```

Comment:
The `===` will return `true` or `false` without the ternary operator.

Reply:
@jkotanchik-SB I made the change.